ee/desktop/notify/notify.go

```diff
@@ -0,0 +1,52 @@
+package notify
```

Comment:
TODO add documentation in this package or in `docs/architecture` covering:
* Why we chose the options we did per-OS for sending notifications
* Why this has to run in the desktop process

Reply:
I'm happy to move it wherever -- I don't have a strong preference for where this lives